src/z3c/menu/ready2go/item.py

```diff
@@ -134,8 +134,8 @@ class SiteMenuItem(MenuItem):
     @property
     def available(self):
         """Available checker call"""
-        root = zope.proxy.getProxiedObject(getRoot(self.context))
-        site = zope.proxy.getProxiedObject(hooks.getSite())
+        root = zope.security.proxy.getObject(getRoot(self.context))
+        site = zope.security.proxy.getObject(hooks.getSite())
         return site is not root
```

Comment:
Any reason not to use `sameProxiedObject()`?

I'm wondering if the proxies involved in this might not necessarily be security proxies, but also location proxies.